doc/_src_docs/examples/b777_engine.rst

```diff
@@ -5,30 +5,30 @@ Boeing 777 engine data set
 
   import numpy as np
   import os
-  
+
```

Comment:
I think the indentation is necessary here for the code block to display correctly? Are you sure about these changes?